openfisca_france_fiscalite_miniere/parameters/redevances/communales/bauxite.yaml

```
@@ -23,3 +23,7 @@ values:
     value: 636.40
     metadata:
       reference: https://www.legifrance.gouv.fr/codes/id/LEGIARTI000043663105/2021-06-12/
+  2022-01-01:
+    value: 671.40
+    metadata:
+      reference: https://www.legifrance.gouv.fr/codes/id/LEGIARTI000045765025/2022-05-07/
```

Ajouter un retour à la ligne pour éviter le 🚫 ?
```suggestion
      reference: https://www.legifrance.gouv.fr/codes/id/LEGIARTI000045765025/2022-05-07/

```